### DISCUSSION TITLE
Imporving the webpage!

### DISCUSSION THREAD

**Discussion by AnkushSinghGandhi** [AnkushSinghGandhi/pomodoro-chrome-extension#16] [Ideas]
Any suggestions to improve the look and feel of the projects' webpage.

**Comment by AnkushSinghGandhi** [comment]
We can add more information about the project and Pomodoro techniques also instead of having a project's demo picture in the middle of the page what we can do is we can have pictures on the left side and text on the right then picture on the right and text on left like this :
#
![Screenshot from 2021-10-25 11-27-11](https://user-images.githubusercontent.com/55637484/138656662-fa5c133c-9951-4f60-9764-4c840d8df205.png)
#
This will look better in my opinion.

**Reply by nikkuv** [reply]
Yeah sure, I will add a section like this. I want to keep that header picture as it will be the main header of the site.

**Reply by GayanKod** [reply]
I think colors need to be changed to get a great look

**Reply by GayanKod** [reply]
Also buttons and hover stylings... feel like need to be more professional

**Comment by AnkushSinghGandhi** [comment]
@nikkuv what's your thought on this?

**Reply by sanyogchangmai** [reply]
I think we can enhance the look if we add animation to text and images, moreover instead of using a picture we can use a gif to demonstrate the working of the timer.

**Reply by nikkuv** [reply]
Yeah, It's a great Idea.
